cmd/analyzers/test-analyzer/main.go

```diff
@@ -23,9 +29,34 @@ func main() {
 		log.Printf("%v failed: %v\n", analyzer.GetModuleName(), err)
 		os.Exit(master.ReturnAnalyzerFailed)
 	}
+	for _, test := range tests {
```

Comment:
Why do you call the tests here? This seems plain wrong.

Reply:
I want to test against Package node. Analyze function is called for each file node. 
Then I should implement another function for that right?

Reply:
Right. Please create an issue for this. Then we should merge this.